SimplyLinkedList.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+
+from Link import Link
```

Comment:
Of course!

In Java, each class is in a file by the same name.

In Python, each file is called a module and can contain multiple classes or functions.
That is because the file itself defines the name of the module. This allows us, among other things, to hide within the module all the implementation details (e.g., the `Link` class here that is not needed by someone using the `SimplyLinkedList`.

You can find of a Python file a little bit like a package in Java, where you can have multiple classes with it.

Usually it is a good rule to hide any implementation details. In Java you could do this with a private nested class (which is accessible only from the class in which it is defined) or a package-private class (which is accessible only from classes in the same package).